app/packages/registration-schema/validations.ts

```diff
@@ -95,6 +95,19 @@ export function validateNodeLicense() {
         if (!validateLicenseTarget || validateLicenseTarget?.requiredFields?.length === 0) {
             return true;
         }
+
+        if (validateLicenseTarget.requiredFields?.includes('year')) {
```

Comment:
This validation may be better with:
` if (validateNodeLicenseTarget && validateLicenseTarget.requiredFields?.includes('year'))`